cmd/aws-cloud-controller-manager/controllers/tagging/tagging-controller.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tagging
+
+import (
+	"k8s.io/klog/v2"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+)
+
+// eksResourceTagPrefix is the prefix for tag to group resources that are used by eks
+// for a particular cluster, this tag is added to the existing tags.
```

Comment:
I know this is just a shell and that this value is only used as a placeholder below. But could you explain what you mean by that this tag will be added to the existing list?

Reply:
We want to add the tag to an existing list if the list is not empty or create a new list with just this tag. For the case of EC2, the controller should add one more tag specified by this controller to the instance and leave the current tags intact.

Reply:
What is the purpose of this tag? It looks very EKS specific, and CCM shouldn't hard code anything that assumes that EKS is being used.

Reply:
+1 to what @olemarkus said. I don't think we want to hardcode anything related to EKS. The controller should take these as inputs and each client (like EKS) using this controller can configure their own tags.

Reply:
Remove the prefix as we want the controller to be generic enough for all users.